src/content/self-hosted/helm-configuration.mdx

```diff
@@ -189,6 +189,23 @@ autoscaler:
   slackWebhook:
 ```
 
+### backend
```

Comment:
`backend` is not a component, not sure where is the best place to document this.
We already have a `globals.jobs` section for settings affecting all the jobs. Maybe moving it to `globals.backend` for settings affecting all the backend services?
Ideally, we should have the `image` section on each backend service, but that's out of the scope of this work.
@ifbyol any ideas?

Reply:
Another option is not to document all these image fields. We only expect people to modify:
- `globals.registry`
- `cli.image`
- `installer.runner`
All other image fields shouldn't be configured by our users

Reply:
It is a good question. `global.backend` could be misleading as BuildKit or Registry are not affected by that, and it is hard to know why some components would be included as `backend` and not others.

I would go with no documenting that section for now.

Reply:
SGTM